Makefile.bindings

```diff
@@ -14,7 +14,13 @@ rpmostree-cxxrs.h: $(binding_rust_sources)
 	  echo cxxbridge failed; exit 1; \
 	fi
 rpmostree-cxxrs.cxx: $(binding_rust_sources) rpmostree-cxxrs.h
-	$(AM_V_GEN) cxxbridge --include rpmostree-cxxrs.h rust/src/lib.rs > $@
+	$(AM_V_GEN) if cxxbridge --include rpmostree-cxxrs.h rust/src/lib.rs > $@.tmp; then \
+	  if test -f $@ && cmp $@.tmp $@ 2>/dev/null; then rm -f $@.tmp; else \
```

Comment:
Optional/follow-up: a comment above `rpmostree-cxxrs.h:` describing why we're doing this `cmp` dance would be helpful.